### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by victorshengwgit** [TwistedW/MIAV#3]
hi, thanks for your excellent work. right now I have a simple question is that how can you get results of others' paper,like your comparison with Ma et al[28], Chen et al[30] in fig.5 ?  have you reproduced their work or ...? I am really appreciate it if you could give me some advice
![Snipaste_2023-06-28_14-46-04](https://github.com/TwistedW/MIAV/assets/87215766/cd3d89ab-eb14-4ebd-9149-ca44c9c2ea84)

**Comment by TwistedW** [comment]
Hello, the two articles mentioned in the picture are open sourced on the AV-DRIVE dataset. You can visualize the results directly on the test set. Of course, you can also directly cite the results from some published papers. Hope it helps you.

**Comment by victorshengwgit** [comment]
Thanks for your reply. Actually, I do not get their training code even inference code, but I really need to get the inference visualization results. So I am wondering if you can offer me the way you got the picture, especially pic c,g as shown in Fig5.

**Comment by TwistedW** [comment]
@victorshengwgit OK, give me your email. I am sending you a vector diagram of this image.

**Comment by victorshengwgit** [comment]
***@***.***







------------------ Original ------------------
From: TwistedW ***@***.***&gt;
Date: Wed,Jun 28,2023 11:01 PM
To: TwistedW/MIAV ***@***.***&gt;
Cc: victorshengwgit ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [TwistedW/MIAV] Questions (Issue #3)

**Comment by TwistedW** [comment]
@victorshengwgit You can email me at twistedwg@hotmail.com.

**Comment by TwistedW** [comment]
Your email address is hidden.

> ***@***.***
> […](#)
> ------------------ Original ------------------ From: TwistedW ***@***.***&gt; Date: Wed,Jun 28,2023 11:01 PM To: TwistedW/MIAV ***@***.***&gt; Cc: victorshengwgit ***@***.***&gt;, Mention ***@***.***&gt; Subject: Re: [TwistedW/MIAV] Questions (Issue #3)

**Comment by weidai00** [comment]
> @victorshengwgit OK, give me your email. I am sending you a vector diagram of this image.

Could you please also send me their all result images to weidaimonash@163.com? Thank you
